pkg/time/fake_image_linux_amd64.go

```diff
@@ -37,7 +38,8 @@ func (it *FakeImage) AttachToProcess(pid int, variables map[string]uint64) error
 		runtime.UnlockOSThread()
 	}()
 
-	program, err := ptrace.Trace(pid)
+	// TODO: Waiting for #2889, use logger provided by FakeImage
```

Comment:
@STRRL we could use logger provided by FakeImage now, according to #2889 is merged.

Reply:
I agree with it.

Reply:
Updated! PTAL!